src/org/yccheok/jstock/gui/JStock.java

```diff
@@ -470,6 +470,11 @@ public void stateChanged(javax.swing.event.ChangeEvent evt) {
         this.jTable1.getTableHeader().addMouseListener(new TableColumnSelectionPopupListener(1));
         this.jTable1.addMouseListener(new TableRowPopupListener());
         this.jTable1.addKeyListener(new TableKeyEventListener());
+        jTable1.addMouseListener(new java.awt.event.MouseAdapter() {
+            public void mouseClicked(java.awt.event.MouseEvent evt) {
+                jTable1MouseClicked(evt);
+            }
+        });
```

Comment:
Since we already have TableRowPopupListener to handle all the mouse click events, should we place "Double click will invoke history chart" logic in single place too for easy maintenance (meaning, there should be only 1 jTable1.addMouseListener call instead of 2) ?

Initially, TableRowPopupListener is designed for handle "showing pop up menu". However, now it is doing more than that, perhaps we should give a new name?